### ISSUE TITLE
Fix calculation of storage space in Disk tab

### ISSUE THREAD

**Issue by elfring** [BOINC/boinc#1389]
I copied my BOINC configuration to an other disk with more free space on an openSUSE system.

```
Filesystem            Size  Used Avail Use% Mounted on
/dev/mapper/doda-usr   55G   42G   12G  79% /usr/vorherige_Montur2
```

I find that the display in the tab "Disk" of the application "BOINC Manager 7.2.42" does not fit to information from a command like "df".

```
Total disk usage
* used by BOINC: 2.31 GB
* free, available to BOINC: 59.82 MB
…
* used by other programs: 34.71 GB
```

The file "global_prefs_override.xml" contains settings like the following.

```
<global_preferences>
…
   <disk_max_used_gb>10.000000</disk_max_used_gb>
   <disk_max_used_pct>99.000000</disk_max_used_pct>
   <disk_min_free_gb>0.010000</disk_min_free_gb>
…
</global_preferences>
```

**Comment by Ageless93** [comment]
Isn't this the same issue as #89?

**Comment by AenBleidd** [comment]
Closed as a duplicate of #89
